RELEASE-NOTES.md

```diff
@@ -1,5 +1,76 @@
+```
+
+Using the `--provision-infra` flag will allow you to select a GCP project to use for your work pool and automatically configure it to be ready to execute flows via Cloud Run:
+
+```
+╭──────────────────────────────────────────────────────────────────────────────────────────────────────────╮
+│ Provisioning infrastructure for your work pool my-pool will require:                                     │
+│                                                                                                          │
+│     Updates in GCP project central-kit-405415 in region us-central1                                      │
+│                                                                                                          │
+│         - Activate the Cloud Run API for your project                                                    │
+│         - Create a service account for managing Cloud Run jobs: prefect-cloud-run                        │
+│             - Service account will be granted the following roles:                                       │
+│                 - Service Account User                                                                   │
+│                 - Cloud Run Developer                                                                    │
+│         - Create a key for service account prefect-cloud-run                                             │
+│                                                                                                          │
+│     Updates in Prefect workspace                                                                         │
+│                                                                                                          │
+│         - Create GCP credentials block my--pool-push-pool-credentials to store the service account key   │
+│                                                                                                          │
+╰──────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+Proceed with infrastructure provisioning? [y/n]: y
+Activating Cloud Run API
+Creating service account
+Assigning roles to service account
+Creating service account key
+Creating GCP credentials block
+Provisioning Infrastructure ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 0:00:00
+Infrastructure successfully provisioned!
+Created work pool 'my-pool'!
+```
+
+If you have yet to try using a Cloud Run run push pool, now is a great time!
+
+If you use another cloud provider, don't fret; we will add support for ECS and Azure Container Instances push work pools in future releases!
+
+See the following pull request for implementation details:
+— https://github.com/PrefectHQ/prefect/pull/11204
+
+### Enhancements
+- Add ability to search for block documents by name — https://github.com/PrefectHQ/prefect/pull/11212
+- Add pagination to the blocks page — https://github.com/PrefectHQ/prefect/pull/11214
```

Comment:
Add pagination to the Blocks page in the Prefect UI for viewing/filtering more than 200 blocks